### ISSUE TITLE
WordPress: Nicepage plugin import failed

### ISSUE THREAD

**Issue by usmannasir** [coreruleset/coreruleset#2317]
### Description

To reproduce this error

Here is the process

1. Install Nicepage plugin (https://nicepage.com/doc/1323/getting-started-with-wp-theme-and-plugin).
4. In the WordPress sidebar, go to Nicepage > Import
5. When uploading a zip file (while choosing **check replace & replace**)
6. Finally after hitting upload we get 403 error.

```
---lDPfp6Zx---H--
ModSecurity: Warning. Matched "Operator `Within' with parameter `/proxy/ /lock-token/ /content-range/ /if/' against variable `TX:header_name_content-range' (Value: `/content-range/' ) [file "/usr/local/lsws/conf/modsec/coreruleset-3.3.2/rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf"] [line "1087"] [id "920450"] [rev ""] [msg "HTTP header is restricted by policy (/content-range/)"] [data "Restricted header detected: /content-range/"] [severity "2"] [ver "OWASP_CRS/3.3.2"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag "attack-protocol"] [tag "paranoia-level/1"] [tag "OWASP_CRS"] [tag "capec/1000/210/272"] [tag "PCI/12.1"] [hostname "domain.com"] [uri "/wp-admin/admin-ajax.php"] [unique_id "1637483735"] [ref "o0,13v1742,13t:lowercaseo0,16v1781,16t:lowercaseo0,7v1574,7t:lowercaseo0,4v1546,4t:lowercaseo0,10v1647,10t:lowercaseo0,14v1927,14t:lowercaseo0,6v266,6t:lowercaseo0,15v74,15t:lowercaseo0,12v141,12t:low (170 characters omitted)"]
ModSecurity: Access denied with code 403 (phase 2). Matched "Operator `Ge' with parameter `5' against variable `TX:ANOMALY_SCORE' (Value: `5' ) [file "/usr/local/lsws/conf/modsec/coreruleset-3.3.2/rules/REQUEST-949-BLOCKING-EVALUATION.conf"] [line "80"] [id "949110"] [rev ""] [msg "Inbound Anomaly Score Exceeded (Total Score: 5)"] [data ""] [severity "2"] [ver "OWASP_CRS/3.3.2"] [maturity "0"] [accuracy "0"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag "attack-generic"] [hostname "domain.com"] [uri "/wp-admin/admin-ajax.php"] [unique_id "1637483735"] [ref ""]

---lDPfp6Zx---Z--
```

### Audit Logs / Triggered Rule Numbers

id "949110"

Seems this one is creating an issue -> https://github.com/coreruleset/coreruleset/blob/4eab4128a7079551551dcea90e1910ce7946d4eb/rules/REQUEST-949-BLOCKING-EVALUATION.conf#L139

### Your Environment

* CRS version (e.g., v3.2.2):
* Paranoia level setting:
* ModSecurity version (e.g., 2.9.3):
* Web Server and version (e.g., apache 2.4.41): OpenLiteSpeed/LiteSpeed Enterprise
* Operating System and version: Ubuntu 20.04

### Confirmation

[ ] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by airween** [comment]
Hi @usmannasir,

thanks for this report. As  you added the label, this is a false positive.

I think the best thing what you can do now is make an exclusion. You can exclude the triggered rule (namely [92450](https://github.com/coreruleset/coreruleset/blob/v3.2/master/rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf#L1027-L1052)), or you can modify the transaction variable [restricted_headers](https://github.com/coreruleset/coreruleset/blob/v3.2/master/rules/REQUEST-901-INITIALIZATION.conf#L197-L203).

Here are some examples:

```
SecRule REQUEST_URI "@streq /wp-admin/admin-ajax.php" \
    "id:100001,\
    phase:1,\
    t:none,\    
    pass,\
    nolog,\
    ctl=ruleRemoveById:920450"
```
or
```
SecRule REQUEST_URI "@streq /wp-admin/admin-ajax.php" \
    "id:100001,\
    phase:1,\
    t:none,\    
    pass,\
    nolog,\
    setvar:'tx.restricted_headers=/proxy/ /lock-token/ /translate/ /if/'"
```
as you can see in the last example the `restricted_headers` gets the values without `content-range`.

Put one of these rules into your REQUEST-900-EXCLUSION-RULES-BEFORE-CRS.conf, and please check the result.

**Comment by usmannasir** [comment]
I will try this and revert to you by tomorrow.

**Comment by usmannasir** [comment]
@airween This seems to resolve the issue.

**Comment by airween** [comment]
@usmannasir - fine, thanks for feedback.

Please close this issue, if you don't have any other question.

**Comment by usmannasir** [comment]
Do you want to keep this opened to resolve the original rule or should I close this ?

**Comment by airween** [comment]
No, we do not want to keep this open - feel free to close it, if you think there is no more question.

**Comment by usmannasir** [comment]
thanks alot.

**Comment by usmannasir** [comment]
Sorry to interrupt again. However this issue is happening again with LiteSpeed Enterprise, even after the provided fix

````
--121523e0-H--
Message: Access denied with code 403 (phase 2). Test 'TX:ANOMALY_SCORE' against '@ge %{tx.inbound_anomaly_score_threshold}' is true. [file "/usr/local/lsws/conf/modsec/coreruleset-3.3.2/rules/REQUEST-949-BLOCKING-EVALUATION.conf"] [line "93"] [id "949110"] [msg "Inbound Anomaly Score Exceeded (Total Score: 5)"] [severity "CRITICAL"] [tag "application-multi"] [tag "language-multi"] [tag "platform-multi"] [tag "attack-generic"] [MatchedString "5"]

--121523e0-C--
```

I've tried to exclude the rule id "949110" , no luck.

**Comment by dune73** [comment]
LiteSpeed Enterprise is a custom ModSecurity engine, but also a customized version of CRS. I think you have to talk to their support, since we do not really know the system.
